src/main/java/org/osc/manager/ism/api/IsmAgentApi.java

```diff
@@ -55,33 +54,28 @@ public static IsmAgentApi create(ApplianceManagerConnectorElement mc, VirtualSys
     @Override
     public List<ManagerDeviceMemberStatusElement> getFullStatus(List<DistributedApplianceInstanceElement> list) {
         List<ManagerDeviceMemberStatusElement> response = new ArrayList<>();
-        if (list != null) {
-            for (DistributedApplianceInstanceElement dai : list) {
-                DeviceMemberEntity member = null;
-                try {
-                    member = (DeviceMemberEntity) this.api.findDeviceMemberByName(dai.getName());
-                } catch (Exception e) {
-                    e.printStackTrace();
-                }
-                if (member != null) {
-                    DeviceMemberStatusEntity memberStatus = new DeviceMemberStatusEntity();
-                    memberStatus.setDistributedApplianceInstanceElement(dai);
-                    memberStatus.setApplianceGateway(member.getApplianceGateway());
-                    memberStatus.setApplianceIp(member.getApplianceIp());
-                    memberStatus.setApplianceSubnetMask(member.getApplianceSubnetMask());
-                    memberStatus.setBrokerIp(member.getBrokerIp());
-                    memberStatus.setManagerIp(member.getManagerIp());
-                    memberStatus.setPublicIp(member.getPublicIp());
-                    memberStatus.setRx(member.getRx());
-                    memberStatus.setTxSva(member.getTxSva());
-                    memberStatus.setDropSva(member.getDropSva());
-                    memberStatus.setVersion(member.getVersion());
-                    memberStatus.setDiscovered(Boolean.TRUE);
-                    memberStatus.setInspectionReady(Boolean.TRUE);
-                    response.add(memberStatus);
-                }
+        if (list == null) {
+            String msg = String.format("Dai is null");
```

Comment:
You dont need string format here since there is no input in the string. Also you dont need to log since you are throwing the exception with the same message. You want to log only if the caller component does not have the necessary information to log it itself.  In this case it does since it receives the exception with exact the same message, logging here just adds redundant log entries. Simply new IllegalArgumentException("The provided DAIs should not be null") would suffice.